### ISSUE TITLE
2.0.0 release tasks

### ISSUE THREAD

**Issue by wesleycho** [angular-ui/bootstrap#5917]
### General
- [ ] Investigate potential to remove `replace: true` in components (#5712)
- [x] Update docs to detail semver policy for 2.0.0 onwards (#5800)

### Datepicker

- [x] Fix ng-model-options for the datepicker (#5825)

### Dropdown

- [ ] Add support for positioning up or down (or auto) with the dropdown (#5712 - dependent on #5918)
- [x] Focus dropdown after selection (#5782)
- [x] Fix keyboard-nav when using with append-to-body (#5965)

### $position
- [x] Refactor the positioning mechanism to include height/width dimensions for `$uibPosition.positionElement` method and remove timeout/deferred positioning logic to fix various edge issues (#5918)

### Tooltip

- [x] Add support for tooltip-trigger (#5221)

### Typeahead
- [x] Remove coupling to `li` tag element with typeahead dropdown (#5848)
- [x] Deduplicate id attribute in typeahead (#5926)

**Comment by icfantv** [comment]
@wesleycho it looks like #5983 is done so is the dropdown checkbox item completed or is that dependent on the `replace: true` issue?  If the latter, we talked about pushing the rest to a subsequent release so we can get 2.0 out the door.  Thoughts?

**Comment by wesleycho** [comment]
Releasing 2.0, moved the remaining tasks to the 3.0.0 milestone
